docs/API.md

```diff
@@ -451,77 +499,146 @@ New version:
+| OptName            | Since | Binding |   Type    | Units  | Default  | Range  | Dir | Entity |
+| ------------------ | ----- | ------- | --------- | ------ | -------- | ------ | --- | ------ |
+| `SRTO_CONNTIMEO`   | 1.1.2 | pre     | `int32_t` | msec   | 3000     | 0..    | W   | GSD+   |
 
-- Connect timeout. SRT cannot connect for RTT > 1500 msec (2 handshake exchanges) 
-with the default connect timeout of 3 seconds. This option applies to the caller 
-and rendezvous connection modes. The connect timeout is 10 times the value set 
-for the rendezvous mode (which can be used as a workaround for this connection 
-problem with earlier versions)
+- Connect timeout. This option applies to the caller and rendezvous connection
+modes. The connect timeout is 10 times the value set for the rendezvous mode
+(which can be used as a workaround for this connection problem with earlier
+versions).
 
 ---
 
-| OptName           | Since | Binding | Type      | Units  | Default  | Range  |
-| ----------------- | ----- | ------- | --------- | ------ | -------- | ------ |
-| `SRTO_EVENT`      |       | n/a     | `int32_t` |        | n/a      | n/a    |
+| OptName           | Since | Binding | Type      | Units  | Default  | Range  | Dir | Entity |
+| ----------------- | ----- | ------- | --------- | ------ | -------- | ------ | --- | ------ |
+| `SRTO_EVENT`      |       |         | `int32_t` | flags  |          |        | R   | S      |
 
-- **[GET]** - Returns bit flags set according to the current active events on 
-the socket. 
+- Returns bit flags set according to the current active events on the socket. 
 - Possible values are those defined in `SRT_EPOLL_OPT` enum (a combination of 
 `SRT_EPOLL_IN`, `SRT_EPOLL_OUT` and `SRT_EPOLL_ERR`).
 
 ---
 
-| OptName               | Since | Binding | Type  | Units  | Default  | Range  |
-| --------------------- | ----- | ------- | ----- | ------ | -------- | ------ |
-| `SRTO_FC`             |       | pre     | `int` | pkts   | 25600    | 32..   |
+| OptName           | Since | Binding | Type      | Units  | Default  | Range  | Dir | Entity |
+| ----------------- | ----- | ------- | --------- | ------ | -------- | ------ | --- | ------ |
+| `SRTO_FC`         |       | pre     | `int32_t` | pkts   | 25600    | 32..   | RW  | GSD    |
 
 - Flight Flag Size (maximum number of bytes that can be sent without 
 being acknowledged)
 
 ---
 
-| OptName               | Since | Binding | Type   | Units  | Default  | Range  |
-| --------------------- | ----- | ------- | ------ | ------ | -------- | ------ |
-| `SRTO_GROUPCONNECT`   |       | pre     | `int`  |        | 0        | 0...1  |
+| OptName              | Since | Binding | Type      | Units  | Default  | Range  | Dir | Entity |
+| -------------------- | ----- | ------- | --------- | ------ | -------- | ------ | --- | ------ |
+| `SRTO_GROUPCONNECT`  |       | pre     | `int32_t` |        | 0        | 0...1  | W   | S      |
```

Comment:
Here is a problem: This option has the earliest version when it was introduced. Needs to be fixed. Probably it should be 1.5.0